### ISSUE TITLE
document opensnoop, iosnoop, and friends under "MacOS Only"?

### ISSUE THREAD

**Issue by jab** [jlevy/the-art-of-command-line#185]
First off, thank you for making this project, it's fantastic!

I recently came across http://dtrace.org/blogs/brendan/2011/10/10/top-10-dtrace-scripts-for-mac-os-x/ (scroll down to the "Scripts" section) and thought you might be interested in adding to your "MacOS Only" section.

Thanks again for the great project.

**Comment by jlevy** [comment]
Yes, these would be a good addition to that section. If you can submit a PR briefly mentioning iosnoop, opensnoop, execsnoop, iotop, and dtruss, I'll add it.
